Formula/arangodb3.rb

```diff
@@ -0,0 +1,85 @@
+class Arangodb3 < Formula
+  desc "The Multi-Model NoSQL Database."
+  homepage "https://www.arangodb.com/"
+  url "https://www.arangodb.com/repositories/Source/ArangoDB-3.0.1.tar.gz"
+  sha256 "f2e9a44bbd73608cace6a3bfe357d2e58aed1eb8b0a4675c36c6a9b15620576a"
+
+  bottle do
+    revision 1
+    sha256 "f9699b28c53f744b7b4701890739d7a9369a6e0b144d486d7fee27ed4b4a9486" => :el_capitan
+    sha256 "7490e2e4026fca6442fce8c6f56d654deb45ba864ca6b7a73872d670b4475d49" => :yosemite
+    sha256 "60d59e09f4a440cffeb077f1de14784519223e6f25ccaaf6df692dc63fd6cac8" => :mavericks
+  end
+
+  head do
+    url "https://github.com/arangodb/arangodb.git", :branch => "unstable"
+    depends_on "cmake" => :build
+  end
+
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+  depends_on "openssl"
+
+  needs :cxx11
+
+  fails_with :clang do
+    build 600
+    cause "Fails with compile errors"
+  end
+
+  def install
+    ENV.libcxx
+
+    mkdir "arangodb-build" do
+      system "cmake", "..", "-DCMAKE_BUILD_TYPE=RelWithDebInfo", "-DHOMEBREW=On", "-DUSE_OPTIMIZE_FOR_ARCHITECTURE=Off", "-DASM_OPTIMIZATIONS=Off", "-DETCDIR=#{prefix}/etc", "-DVARDIR=#{var}", *std_cmake_args
+      system "make", "install"
+    end
+  end
+
+  # moving the "if" inside post_install does not work
+  def post_install
+    (var/"lib/arangodb3").mkpath
+    (var/"log/arangodb3").mkpath
+  end
+
+  def caveats
+    s = <<-EOS.undent
+      The database format between ArangoDB 2.x and ArangoDB 3.x has
+      been changed. Please export the old database using arangodump
+      and import it again using arangorestore.
+
+      An empty password has been set. Please change it by executing
+        #{sbin}/arango-secure-installation
+    EOS
+
+    s
+  end
+
+  plist_options :manual => "#{HOMEBREW_PREFIX}/opt/arangodb3/sbin/arangod"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <true/>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_sbin}/arangod</string>
+          <string>-c</string>
+          <string>#{etc}/arangodb3/arangod.conf</string>
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    assert_equal "it works!\n", shell_output("#{bin}/arangosh --javascript.execute-string \"require('@arangodb').print('it works!')\"")
```

Comment:
would be good if this could be refactored into multiple lines so it doesn't exceed the 80 character column limit